solutions/control-flow/clock.js

```diff
@@ -0,0 +1,21 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+let s = readLine().split(" ");
+let n1 = parseInt(s[0]);
```

Comment:
@Devjyoti29  Please use valid namings for n1,n2,s. And also in description it is given like num1,num2 for numbers.